trading_bot.py

```
@@ -2,24 +2,23 @@
 Simple Kalshi trading bot with Octagon research and OpenAI decision making.
 """
 import asyncio
-import json
-import re
-import sys
 import argparse
-from typing import List, Dict, Any, Optional
-from loguru import logger
+import json
+import csv
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)